PyQt5-stubs/QtNetwork.pyi

```diff
@@ -764,13 +764,13 @@ class QNetworkAccessManager(QtCore.QObject):
     def setProxyFactory(self, factory: 'QNetworkProxyFactory') -> None: ...
     def proxyFactory(self) -> 'QNetworkProxyFactory': ...
     def createRequest(self, op: 'QNetworkAccessManager.Operation', request: 'QNetworkRequest', device: typing.Optional[QtCore.QIODevice] = ...) -> 'QNetworkReply': ...
-    def preSharedKeyAuthenticationRequired(self, reply: 'QNetworkReply', authenticator: 'QSslPreSharedKeyAuthenticator') -> None: ...
+    preSharedKeyAuthenticationRequired = typing.ClassVar[QtCore.pyqtSignal]
     def networkAccessibleChanged(self, accessible: 'QNetworkAccessManager.NetworkAccessibility') -> None: ...
```

Comment:
[`networkAccessibleChanged`](https://doc.qt.io/qt-5/qnetworkaccessmanager-obsolete.html#networkAccessibleChanged) is a signal too, just deprecated/obsolete. I think we should fix it as well

Reply:
You're right, haven't thought about deprecated signals.